Code/Framework/AzCore/AzCore/Asset/AssetManager.cpp

```diff
@@ -2232,11 +2230,17 @@ namespace AZ::Data
 
 #ifdef AZ_ENABLE_TRACING
         auto loadMs = AZStd::chrono::duration_cast<AZStd::chrono::milliseconds>(
-                      AZStd::chrono::system_clock::now() - start);
-        AZ_Warning("AssetDatabase", (!cl_assetLoadWarningEnable) ||
-                   loadMs <= AZStd::chrono::milliseconds(cl_assetLoadWarningMsThreshold),
-                   "Load time threshold exceeded: LoadAssetData call for %s took %" PRId64 " ms",
-                   asset.GetHint().c_str(), loadMs.count());
+            AZStd::chrono::system_clock::now() - start);
+        if (loadMs.count() > 0)
+        {
+            const double seconds = loadMs.count() / 1000.0;
+            const double kilobytes = stream->GetLoadedSize() / 1024.0;
+            const double rateKbps = kilobytes / seconds;
+            AZ_Warning("AssetDatabase", (!cl_assetLoadWarningEnable) ||
+                loadMs <= AZStd::chrono::milliseconds(cl_assetLoadWarningMsThreshold),
+                "Load time threshold exceeded: LoadAssetData call for %s took %" PRId64 " ms %8.4Lf KB/s rate %" PRId64 " bytes",
```

Comment:
Suggestion: Since you're now showing total bytes, milliseconds, and KB/sec, it might read a little better like this:
```
"Load time threshold exceeded: LoadAssetData call for %s took %" PRId64 " ms to load %" PRId64 " bytes (%8.4Lf KB/s)"
```